tests/test_commands/data/static_test_data.py

```diff
@@ -172,6 +179,15 @@ def _load_static_data(self):
             }
         )
 
+        self._onpremises_gateway_details = OnPremisesGatewayDetails(
+            {
+                "id": get_env_with_default("FABRIC_CLI_TEST_ONPREMISES_GATEWAY_ID"),
+                "encrypted_credentials": get_env_with_default(
+                    "FABRIC_CLI_TEST_ONPREMISES_GATEWAY_ENCRIPTED_CRADENTIALS"
```

Comment:
If this is a new key, please ensure its value is added in relevant internal location for the development team.